examples/experimental/dagster-airlift/dagster_airlift_tests/airflow_migrating_project/dags/simple_dag.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from dagster_airlift import mark_as_dagster_migrating
+
+
+def print_hello():
+    print("Hello")  # noqa: T201
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2023, 1, 1),
+    "retries": 1,
+}
+
+dag = DAG(
+    "print_dag", default_args=default_args, schedule_interval=None, is_paused_upon_creation=False
+)
+print_op = PythonOperator(task_id="print_task", python_callable=print_hello, dag=dag)
+downstream_print_op = PythonOperator(
+    task_id="downstream_print_task", python_callable=print_hello, dag=dag
+)
+
+mark_as_dagster_migrating(
+    {
+        "print_dag": {
+            "print_task": True,
+            "downstream_print_task": False,
+        }
+    }
+)
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +27 to +35]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->does this dictionary have real meaning or is it just metadata? If it actually changes behavior I think it should be a proper class with a docstring etc<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23370?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
On the airflow side, it's just metadata. However, it could change behavior on the dagster side.